ray-operator/test/e2e/support.go

```diff
@@ -175,3 +178,54 @@ func jobSubmitterPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyC
 						corev1.ResourceMemory: resource.MustParse("500Mi"),
 					}))))
 }
+
+func IsEnvVarExisted(env string, pod *corev1.Pod) bool {
```

Comment:
This function doesn’t appear to be used in other packages, so we don’t need to export it. That is, use a lowercase `i` instead of an uppercase `I`.